packages/base-zone/README.md

```diff
@@ -0,0 +1,9 @@
+# Base Zone Library
+
+This library is used internally by [`@agoric/zone`](../zone/README.md); refer to it for more details.  Unless you are an author of a new Zone backing store type, or want to use `makeHeapZone` without introducing build dependencies on `@agoric/vat-data`, you should instead use `@agoric/zone`.
```

Comment:
Consider to flip the ordering to begin with what the package offers. Then why you would or wouldn't want to use it directly.